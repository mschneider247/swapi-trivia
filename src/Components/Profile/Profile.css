.profile {
  display: flex;
  flex-direction: column;
  margin-left: 40px;
  text-align: left;
}

.profile-p {
  color: #FFFFFF;
  letter-spacing: 5px;
  margin: 5px 0;
}
