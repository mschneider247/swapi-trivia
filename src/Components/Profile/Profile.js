import React from 'react';
import './Profile.css';

const Profile = ( { name, quote, status } ) => {
  return (
    <section className="profile">
      <p><span className="bold-text">NAME </span>{name}</p>
      <p><span className="bold-text">QUOTE </span>{quote}</p>
      <p><span className="bold-text">STATUS </span>{status}</p>
    </section>
  )
}

export default Profile;
