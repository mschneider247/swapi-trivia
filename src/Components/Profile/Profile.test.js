import React from 'react';
import ReactDOM from 'react-dom';
import { shallow } from 'enzyme';
import Profile from './Profile';

describe('Profile', () => {
  it('should match the snapshot with correct data', () => {

    const wrapper = shallow(
      <Profile
        name="Mike"
        quote="Hey"
        status="YODA"
      />
    );

    expect(wrapper).toMatchSnapshot()

  })
})
