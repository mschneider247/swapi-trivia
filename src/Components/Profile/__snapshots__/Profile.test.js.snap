// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Profile should match the snapshot with correct data 1`] = `
<section
  className="profile"
>
  <p
    className="profile-p"
  >
    <span
      className="bold-text"
    >
      NAME 
    </span>
    Mike
  </p>
  <p
    className="profile-p"
  >
    <span
      className="bold-text"
    >
      QUOTE 
    </span>
    Hey
  </p>
  <p
    className="profile-p"
  >
    <span
      className="bold-text"
    >
      STATUS 
    </span>
    YODA
  </p>
</section>
`;
